src/fields/OpenBoundary.H

```diff
@@ -0,0 +1,194 @@
+    s_v*(pow<5>(x)*y - 10.0_rt/3.0_rt*pow<3>(x)*pow<3>(y) + x*pow<5>(y)),
+    s_v*(pow<7>(x) - 21*pow<5>(x)*pow<2>(y) + 35*pow<3>(x)*pow<4>(y) - 7*x*pow<6>(y)),
+    s_v*(-7*pow<6>(x)*y + 35*pow<4>(x)*pow<3>(y) - 21*pow<2>(x)*pow<5>(y) + pow<7>(y)),
+    s_v*(pow<8>(x) - 28*pow<6>(x)*pow<2>(y) + 70*pow<4>(x)*pow<4>(y) - 28*pow<2>(x)*pow<6>(y) + pow<8>(y)),
+    s_v*(-pow<7>(x)*y + 7*pow<5>(x)*pow<3>(y) - 7*pow<3>(x)*pow<5>(y) + x*pow<7>(y)),
+    s_v*(pow<9>(x) - 36*pow<7>(x)*pow<2>(y) + 126*pow<5>(x)*pow<4>(y) - 84*pow<3>(x)*pow<6>(y) + 9*x*pow<8>(y)),
+    s_v*(9*pow<8>(x)*y - 84*pow<6>(x)*pow<3>(y) + 126*pow<4>(x)*pow<5>(y) - 36*pow<2>(x)*pow<7>(y) + pow<9>(y)),
+    s_v*(-pow<10>(x) + 45*pow<8>(x)*pow<2>(y) - 210*pow<6>(x)*pow<4>(y) + 210*pow<4>(x)*pow<6>(y) - 45*pow<2>(x)*pow<8>(y) + pow<10>(y)),
+    s_v*(pow<9>(x)*y - 12*pow<7>(x)*pow<3>(y) + (126.0_rt/5.0_rt)*pow<5>(x)*pow<5>(y) - 12*pow<3>(x)*pow<7>(y) + x*pow<9>(y)),
+    s_v*(pow<11>(x) - 55*pow<9>(x)*pow<2>(y) + 330*pow<7>(x)*pow<4>(y) - 462*pow<5>(x)*pow<6>(y) + 165*pow<3>(x)*pow<8>(y) - 11*x*pow<10>(y)),
+    s_v*(-11*pow<10>(x)*y + 165*pow<8>(x)*pow<3>(y) - 462*pow<6>(x)*pow<5>(y) + 330*pow<4>(x)*pow<7>(y) - 55*pow<2>(x)*pow<9>(y) + pow<11>(y)),
+    s_v*(pow<12>(x) - 66*pow<10>(x)*pow<2>(y) + 495*pow<8>(x)*pow<4>(y) - 924*pow<6>(x)*pow<6>(y) + 495*pow<4>(x)*pow<8>(y) - 66*pow<2>(x)*pow<10>(y) + pow<12>(y)),
+    s_v*(-pow<11>(x)*y + (55.0_rt/3.0_rt)*pow<9>(x)*pow<3>(y) - 66*pow<7>(x)*pow<5>(y) + 66*pow<5>(x)*pow<7>(y) - 55.0_rt/3.0_rt*pow<3>(x)*pow<9>(y) + x*pow<11>(y)),
+    s_v*(pow<13>(x) - 78*pow<11>(x)*pow<2>(y) + 715*pow<9>(x)*pow<4>(y) - 1716*pow<7>(x)*pow<6>(y) + 1287*pow<5>(x)*pow<8>(y) - 286*pow<3>(x)*pow<10>(y) + 13*x*pow<12>(y)),
+    s_v*(13*pow<12>(x)*y - 286*pow<10>(x)*pow<3>(y) + 1287*pow<8>(x)*pow<5>(y) - 1716*pow<6>(x)*pow<7>(y) + 715*pow<4>(x)*pow<9>(y) - 78*pow<2>(x)*pow<11>(y) + pow<13>(y)),
+    s_v*(-pow<14>(x) + 91*pow<12>(x)*pow<2>(y) - 1001*pow<10>(x)*pow<4>(y) + 3003*pow<8>(x)*pow<6>(y) - 3003*pow<6>(x)*pow<8>(y) + 1001*pow<4>(x)*pow<10>(y) - 91*pow<2>(x)*pow<12>(y) + pow<14>(y)),
+    s_v*(pow<13>(x)*y - 26*pow<11>(x)*pow<3>(y) + 143*pow<9>(x)*pow<5>(y) - 1716.0_rt/7.0_rt*pow<7>(x)*pow<7>(y) + 143*pow<5>(x)*pow<9>(y) - 26*pow<3>(x)*pow<11>(y) + x*pow<13>(y)),
+    s_v*(pow<15>(x) - 105*pow<13>(x)*pow<2>(y) + 1365*pow<11>(x)*pow<4>(y) - 5005*pow<9>(x)*pow<6>(y) + 6435*pow<7>(x)*pow<8>(y) - 3003*pow<5>(x)*pow<10>(y) + 455*pow<3>(x)*pow<12>(y) - 15*x*pow<14>(y)),
+    s_v*(-15*pow<14>(x)*y + 455*pow<12>(x)*pow<3>(y) - 3003*pow<10>(x)*pow<5>(y) + 6435*pow<8>(x)*pow<7>(y) - 5005*pow<6>(x)*pow<9>(y) + 1365*pow<4>(x)*pow<11>(y) - 105*pow<2>(x)*pow<13>(y) + pow<15>(y)),
+    s_v*(pow<16>(x) - 120*pow<14>(x)*pow<2>(y) + 1820*pow<12>(x)*pow<4>(y) - 8008*pow<10>(x)*pow<6>(y) + 12870*pow<8>(x)*pow<8>(y) - 8008*pow<6>(x)*pow<10>(y) + 1820*pow<4>(x)*pow<12>(y) - 120*pow<2>(x)*pow<14>(y) + pow<16>(y)),
```

Comment:
Was this generated by human-readable Python code or sth? If so, can the script be added in `hipace/src/utils/`?

Reply:
Done